### PR TITLE
[chore] simplify a couple tests

### PR DESCRIPTION
**Pull request by benmccann**
This is a simpler way to write these tests which are flaking. No idea whether it will help or not, but worth a try

### CODE REVIEW

**Comment by benmccann** [comment]
darn. didn't work

File: packages/kit/test/apps/basics/test/test.js [hunk 1]
```diff
@@ -1909,10 +1903,7 @@ test.describe('Actions', () => {
 		expect(await page.textContent('pre.formdata2')).toBe(JSON.stringify(null));
 
 		await page.type('input[name="username"]', 'foo');
```
Comment (ambiguous48) [inline packages/kit/test/apps/basics/test/test.js:1905]: @benmccann Is there a specific reason we're using [`type`](https://playwright.dev/docs/api/class-locator#locator-type) not [`fill`](https://playwright.dev/docs/api/class-locator#locator-fill)?

`fill` seems more robust, but I wasn't sure if we were deliberately not using it precisely _because_ of the extra checks it carries out.

Comment (ambiguous48) [reply packages/kit/test/apps/basics/test/test.js:1905]: It's also used in some of other flaky tests here so I'm thinking it could be a culprit.

Comment (benmccann) [reply packages/kit/test/apps/basics/test/test.js:1905]: I didn't write this test, so I'm not sure. I don't know why `fill` would be more robust, but at the same time this test is rather flaky so maybe it doesn't hurt to try it

**Pull request closed without merging**